challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |str| str.capitalize }
 end
```

Comment:
In cases like this where you just want to pass everything in an enumerable through a single function call, there's a cuter syntax: `input.map(&:capitalize)`. Check that out and look into why it works. =) But overall good job! :+1: